episodes/4-minimal-reproducible-code.Rmd

```diff
@@ -137,7 +145,48 @@ counts_per_day %>%
        y = "Individuals per day",
        fill = "Species")
 
+### ***EDIT-start *** ###
+
+# If I were to try and attempt to plot "krats per day, per plot this is what I would do:
+counts_per_day %>%
+  ggplot(aes(x=plot_type, y=count_per_day, fill=species_id)) +
+  geom_boxplot() 
+
+# what is the group=interaction() adding?
+
+# I feel like we need to figure out what these exclosures are doing and if it's an over-time kinda thing then that needs to be accounted for. This just doesn't look right to me?
```

Comment:
Absolutely. I was trying to keep the questions reeeeeealllly simple here because we just don't have the space to dive into investigating the biological dynamics in a lot of detail. We can discuss changing the research question to a more temporal one if you think that would be more easily understandable to illustrate the problems!

Reply:
I was just thinking we should double-check what the exclosures were doing within the time frame we are looking at, because if they originally weren't exclosures then of course they weren't working, which could be a nugget that we add later too. I just want to make sure we get the biological interpretation of the data right, if that makes sense? And to make sure there is continuity across the episodes. I don't remember what didn't look right right now, I'll look at it again in the rendered version